core/src/main/java/org/mule/runtime/core/api/extension/MuleExtensionModelDeclarer.java

```diff
@@ -572,6 +573,47 @@ private void declareScatterGather(ExtensionDeclarer extensionDeclarer, ClassType
     // ConstructModel doesn't support it.)
   }
 
+  private void declareSplitAggregate(ExtensionDeclarer extensionDeclarer, ClassTypeLoader typeLoader) {
```

Comment:
are you missing the `collection` parameter here?

Reply:
yes

Reply:
why? add a comment

Reply:
I was missing it. Added it now :-)